src/lib/sort-rules.js

```diff
@@ -1,9 +1,27 @@
 'use strict'
 
 function getSortedRules(rules) {
-  return rules.sort(function sort(a, b) {
-    return a > b ? 1 : -1
-  })
+  return Array.isArray(rules) ? getSortedRulesArray(rules) : transformIntoSortedRulesArray(rules)
+}
+
+function getSortedRulesArray(rules) {
+  return rules.sort(sortAlphabetically)
+}
+
+function transformIntoSortedRulesArray(rules) {
+  var sortedRules = []
+
+  Object.keys(rules)
+    .sort(sortAlphabetically)
+    .forEach(function map(ruleName) {
+      sortedRules.push(ruleName, rules[ruleName].config1, rules[ruleName].config2)
```

Comment:
I guess, now I understand

[`['bar-rule', '1', '2', 'baz-rule', '3', '4']`](https://github.com/sarbbottam/eslint-find-rules/pull/50/files#diff-4ccdee5ead7dda1f5f82d46aefaa5ef2R23)

Looks like, `bin/diff.js` expects this data-structure, I am yet to go through the `bin/diff.js`

If yes, I would request to use non-positional schema.